src/dc_aheader.rs

```diff
@@ -257,18 +260,19 @@ pub unsafe fn dc_aheader_render(mut aheader: *const dc_aheader_t) -> *mut libc::
             &mut ret,
             b"keydata= \x00" as *const u8 as *const libc::c_char,
         );
+        // TODO replace 78 with enum /rtn
         /* adds a whitespace every 78 characters, this allows libEtPan to wrap the lines according to RFC 5322
         (which may insert a linebreak before every whitespace) */
         keybase64_wrapped = dc_key_render_base64(
             (*aheader).public_key,
-            78i32,
+            78,
             b" \x00" as *const u8 as *const libc::c_char,
-            0i32,
+            0,
         );
         if !keybase64_wrapped.is_null() {
             /*no checksum*/
             dc_strbuilder_cat(&mut ret, keybase64_wrapped);
-            success = 1i32
+            success = 1
```

Comment:
@dignifiedquire Why isn't there a `;` at the end here?

Reply:
the converter left them out in some places where they were not a 100% required